src/mero_cluster.erl

```diff
@@ -346,10 +347,10 @@ worker_by_index_function(WorkerDefs) ->
     lists:flatten(
         lists:foldr(
             fun
-                ({Name, ShardSizeAcc, ReplicationNumber, {_Name, _Host, _Port, WorkerName, _WorkerModule}}, []) ->
+                ({Name, ShardSizeAcc, ReplicationNumber, {_, _, _, WorkerName, _}}, []) ->
```

Comment:
I actually think it is good practice to keep the names of the variables so you know what they are should you need them in the future

Reply:
Yeah. this was a _hacky_ way to get below 100 characters, sorry 🙈